sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/odbc.yaml

```diff
@@ -183,8 +183,26 @@ provider:
         post:
           # Avoid matching values found on SourceGraph that look like dummy passwords or insertions.
           patternNot:
-            - "^(?i)pass(word|wd)?(?-i)$|^(?i)pwd(?-i)$|^\\[?PLACEHOLDER\\]?$|^%s$|^\\*{3,}$|^\\{\\d*\\}$|^\\#\\{|^\\{{2,}"
+            - "^(?i)pass(word|wd)$"
```

Comment:
word or wd are not optional anymore. Is this change intended?

Reply:
correct !
```suggestion
            - "^(?i)pass(word|wd)?$"
```